mdtraj/geometry/shape.py

```diff
@@ -126,7 +126,7 @@ def acylindricity(traj):
     c = pm[:,1] - pm[:,0]
     return c
 
-def relative_shape_antisotropy(traj):
+def relative_shape_anisotropy(traj):
     """Compute the acylindricity of a trajectory.

```

Comment:
All the changes look good, but the docs here still imply that the function computes the acylindricity instead of the relative shape anisotropy.